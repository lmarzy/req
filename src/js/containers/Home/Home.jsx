import React, { Component } from 'react';

class Home extends Component {
  constructor(props) {
    super(props);

    this.state = {
      siteName: 'Req',
    };
  }

  render() {
    return (
      <h2>{this.state.siteName}</h2>
    );
  }
}

export default Home;
