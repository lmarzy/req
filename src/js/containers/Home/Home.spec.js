import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Home from './Home';

const component = shallow(<Home />);
const tree = renderer.create(<Home />).toJSON();

describe('<Home />', () => {
  it('Should render 1 <Home /> component', () => {
    expect(component).toHaveLength(1);
  });

  it('Should render a snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
