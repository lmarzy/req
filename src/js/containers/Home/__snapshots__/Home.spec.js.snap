// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Should render a snapshot 1`] = `
<div>
  <h1>
    REQ
  </h1>
</div>
`;
